manifests/init.pp

```diff
@@ -5,6 +5,9 @@
 # @param nameservers
 #   Array of nameservers. The default use Google's public name servers.
 #
+# @param nameserver_limit
+#   Integer of the number of nameservers to allow in the resolv.conf
```

Comment:
Please state that if you go over the limit, that the first X nameservers will be preserved and trailing ones will be discarded.